utils/aws/s3.ts

```diff
@@ -0,0 +1,29 @@
+import { s3 } from "aws/index";
+import fs from "fs";
+import path from "path";
+
+/**
+ * Uploads a file to the AWS S3 bucket corresponding to env.AWS_BUCKET_NAME
```

Comment:
nit: change comment because the bucket doesn't correspond to `AWS_BUCKET_NAME` anymore from the perspective of this file

Reply:
did you do this ?